alpine/alpine_test.go

```diff
@@ -1,43 +1,42 @@
-package alpine
+package alpine_test
```

Comment:
I like the idea of keeping the tests in the same package as the package itself. One advantage I see with it is not having an extra import for a package and being able to test un-exported stuff inside.

Reply:
At first, this is a well-known technique. Many Go standard packages use this technique.
https://github.com/golang/sys/blob/master/unix/mmap_unix_test.go

I think this has some advantages.
https://segment.com/blog/5-advanced-testing-techniques-in-go/

The biggest advantage is we have a chance to think if an interface is appropriate and easy-to-use.